### PR TITLE
docs: fix formatting public dashboards

### PR DESCRIPTION
**Pull request by imatwawana**
Fix formatting of headings and change bullet lists to numbered lists where the items are steps

TO DO
- [x] Confirm which version of docs this should apply to. Setting as no-backport for now. - Last set of changes only in "next" so no backport.

### CODE REVIEW

File: docs/sources/dashboards/dashboard-public/index.md [hunk 1]
```diff
@@ -40,34 +40,36 @@ If you are using Docker, use an environment variable to enable public dashboards
...
-#### Make a dashboard public
+## Make a dashboard public
 
-- Click on the sharing icon to the right of the dashboard title.
-- Click on the **Public dashboard** tab.
-- Acknowledge the implications of making the dashboard public by checking all the checkboxes.
-- Click **Generate public URL** to make the dashboard public and make your link live.
-- Copy the public dashboard link if you'd like to share it. You can always come back later for it.
+1. Click on the sharing icon to the right of the dashboard title.
+1. Click on the **Public dashboard** tab.
```
Comment (Eve832) [inline docs/sources/dashboards/dashboard-public/index.md:46]:
```suggestion
1. Click the sharing icon to the right of the dashboard title.
1. Click the **Public dashboard** tab.
```

File: docs/sources/dashboards/dashboard-public/index.md [hunk 2]
```diff
@@ -40,34 +40,36 @@ If you are using Docker, use an environment variable to enable public dashboards
...
 
-#### Pause access
+## Pause access
 
-- Click on the sharing icon to the right of the dashboard title.
-- Click on the **Public dashboard** tab.
-- Enable the **Pause sharing dashboard** toggle.
-- The dashboard is no longer accessible, even with the link, until you make it shareable again.
+1. Click on the sharing icon to the right of the dashboard title.
+1. Click on the **Public dashboard** tab.
```
Comment (Eve832) [inline docs/sources/dashboards/dashboard-public/index.md:54]:
```suggestion
1. Click the sharing icon to the right of the dashboard title.
1. Click the **Public dashboard** tab.
```

File: docs/sources/dashboards/dashboard-public/index.md [hunk 3]
```diff
@@ -40,34 +40,36 @@ If you are using Docker, use an environment variable to enable public dashboards
...
-- Click on the sharing icon to the right of the dashboard title.
-- Click on the **Public dashboard** tab.
-- Click **Revoke public URL** to delete the public dashboard.
-- The link no longer works. You must create a new public URL as in [Make a dashboard public](#make-a-dashboard-public).
+## Revoke access
 
-#### Supported datasources
+1. Click on the sharing icon to the right of the dashboard title.
+1. Click on the **Public dashboard** tab.
+1. Click **Revoke public URL** to delete the public dashboard.
```
Comment (Eve832) [inline docs/sources/dashboards/dashboard-public/index.md:63]:
```suggestion
1. Click the sharing icon to the right of the dashboard title.
1. Click the **Public dashboard** tab.
1. Click **Revoke public URL** to delete the public dashboard.
```

**Review by Eve832 — APPROVED** [review]

File: docs/sources/dashboards/dashboard-public/index.md (hunk 1 above)
Comment (imatwawana) [reply docs/sources/dashboards/dashboard-public/index.md:46]: I plan to circle back and do a copy edit on a different PR, but just wanted to get the formatting stuff fixed first.

**Pull request merged**